packages/nextra/src/client/pages.ts

```diff
@@ -7,7 +7,10 @@ import { logger } from '../server/utils.js'
 export async function importPage(pathSegments: string[] = [], lang = '') {
   const RouteToFilepath = await getRouteToFilepath(lang)
 
-  const pagePath = RouteToFilepath[pathSegments.join('/')]
+  const path = pathSegments.join('/')
+  const decodePath = decodeURI(path) // handle non-"\w" characters
```

Comment:
```suggestion
  const pathname = pathSegments.join('/')
  const decodePath = decodeURI(pathname) // handle non-"\w" characters
```
to avoid confusing with Node.js' `path` module